tests/post/features/pods_alive.feature

```diff
@@ -0,0 +1,6 @@
+@liveness @post @ci @local
```

Comment:
We must somewhere document the various tags/categories.

Reply:
Yes agree, I just wrote them for testing purposes but the fact that we can filter tests based on tags is so helpful (like @ci or @local)